CHANGELOG.md

```diff
@@ -80,6 +80,7 @@ The order, collapsed state of columns and rows limit is now persisted across pag
 - server: `type` field is not required if `jwk_url` is provided in JWT config
 - server: add a new field `claims_namespace_path` which accepts a JSON Path for looking up hasura claim in the JWT token (#4349)
 - server: support reusing Postgres scalars in custom types (close #4125)
+- server: start-up configuration for sending debug information with errors (close #4031)
```

Comment:
Could you take this to the highlights section? :)

And some more additional information about debugging mode including this [link](https://github.com/rakeshkky/graphql-engine/blob/issue-4031-actions-webhook-debug/docs/graphql/manual/deployment/graphql-engine-flags/config-examples.rst#debugging-mode) (Will fix with prod link in release notes)

Reply:
The breaking change should also be highlighted in the changelog

Reply:
@tirumaraiselvan I'm happy to say that this PR isn't a breaking change any more. :smile:

Reply:
@rakeshkky I'm happy to hear that.